### ISSUE TITLE
changing frequency without mouse

### ISSUE THREAD

**Issue by renerrrr** [F1ATB/Remote-SDR#24]
When using a (common) touchpad on a laptop and no mouse, it is very difficult to change the frequency. It is possible to configure a 'scrollfunction' on a laptop but it's not standard. It would be great if there is an alternative to change the frequency.

**Comment by F1ATB** [comment]
I agree that with a touchpad, it is not easy to change the frequency. Nevertheless, there is an alternative with the left and right arrow of the keyboard.

**Comment by F1ATB** [comment]
You can also construct a rotary knob as described here:
[https://f1atb.fr/index.php/2021/12/06/rotary-knob-and-morse-manipulator-for-remote-sdr/](https://f1atb.fr/index.php/2021/12/06/rotary-knob-and-morse-manipulator-for-remote-sdr/)

**Comment by F1ATB** [comment]
In version 5.07, I have intoduced new features to change the frequency. Go on "Tools" page and click on "Look for updates".
73
F1ATB
